staging/src/k8s.io/client-go/util/cert/cert.go

```diff
@@ -188,10 +188,10 @@ func GenerateSelfSignedCertKeyWithFixtures(host string, alternateIPs []net.IP, a
 	}
 
 	if len(fixtureDirectory) > 0 {
-		if err := os.WriteFile(certFixturePath, certBuffer.Bytes(), 0644); err != nil {
+		if err := os.WriteFile(certFixturePath, certBuffer.Bytes(), 0600); err != nil {
```

Comment:
we don't know what else may be relying on being able to read certificate files... leaving those as 644 seems more reasonable to me

Reply:
agree, we can change permissions only for keys